server/src/modules/users/infra/repos/auth-secret-repo/implementations/mock-auth-secret-repo.ts

```diff
@@ -9,33 +9,32 @@ export class MockAuthSecretRepo implements AuthSecretRepo {
   protected authSecretEntities: Map<string, AuthSecretEntity>
 
   constructor(authSecretEntities: Array<AuthSecretEntity> = []) {
-    this.authSecretEntities = new Map(authSecretEntities.map((authSecretEntity) => 
-      [authSecretEntity.clientId, authSecretEntity]
-    ))
+    this.authSecretEntities = new Map(
+      authSecretEntities.map((authSecretEntity) => [authSecretEntity.clientId, authSecretEntity])
+    )
   }
 
   async exists(clientId: string): Promise<Result<boolean, DBErrors>> {
     for (const authSecretEntity of this.authSecretEntities.values()) {
       if (authSecretEntity.clientId === clientId) return Result.ok(true)
     }
-    return Result.err(new DBError.AuthSecretNotFoundError(clientId))
+    return Result.ok(false)
```

Comment:
this is a mock repo so performance doesn't matter, but you could actually implement it in O(1) time with `return Result.ok(this.authSecretEntities.has(clientId))`